src/license_text_normalizer/main.py

```diff
@@ -147,7 +148,8 @@ def _strip_words(line: str, words_to_strip: List[str]) -> str:
     return normalized_line.lstrip()
 
 
-def _is_repeated_non_alnum(line: str) -> bool:
-    if not len(line) > 1:
+def _is_line_non_alnum(line: str, delimiters: List[str]) -> bool:
+    if not (len(line) > 1 and line[0] in delimiters):
         return False
-    return not line[0].isalnum() and line == len(line) * line[0]
+    line_without_spaces = line.replace(" ", "")
```

Comment:
It's my understanding that spaces are considered to be non-alphanumeric, and so we shouldn't need to strip them out before performing our check.